anyway/models.py

```diff
@@ -331,7 +332,17 @@ def bounding_box_query(is_thin=False, yield_per=None, involved_and_vehicles=Fals
         if kwargs.get('surface', 0) != 0:
             markers = markers.filter(AccidentMarker.road_surface == kwargs['surface'])
         if kwargs.get('acctype', 0) != 0:
-            markers = markers.filter(AccidentMarker.subtype == kwargs['acctype'])
+            if kwargs['acctype'] <= 20:
+                markers = markers.filter(AccidentMarker.subtype == kwargs['acctype'])
+            elif kwargs['acctype'] == 21:
+                markers = markers.\
+                    filter(AccidentMarker.vehicle.any(Vehicle.vehicle_type == 15)).group_by(AccidentMarker.id).\
```

Comment:
I think these numeric constants should definitely be defined somewhere, at least as Python constants. This is true regarding the other constants in thsi function, but at least let's do it for new code.